patches/@changesets+apply-release-plan+6.1.2.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.cjs.dev.js b/node_modules/@changesets/apply-release-plan/dist/apply-release-plan.cjs.dev.js
-index 4c476cb..662448e 100644
+index 0c4d28e..aac0d8d 100644
```

Comment:
[note] I've changed the patch file by running `npx patch-package @changesets/apply-release-plan` (29b079585d018e1a1da500fc25f208ed3aa608a4).

This modification is to remove the following waring of `patch-package` when running `npm install`:

```
Warning: patch-package detected a patch file version mismatch

  Don't worry! This is probably fine. The patch was still applied
  successfully. Here's the deets:

  Patch file created for

    @changesets/apply-release-plan@6.1.0

  applied to

    @changesets/apply-release-plan@6.1.1

  At path

    node_modules/@changesets/apply-release-plan

  This warning is just to give you a heads-up. There is a small chance of
  breakage even though the patch was applied successfully. Make sure the package
  still behaves like you expect (you wrote tests, right?) and then run

    patch-package @changesets/apply-release-plan

  to update the version in the patch file name and make this warning go away.

---
patch-package finished with 1 warning(s).
```